main.tf

```diff
@@ -26,7 +26,8 @@ data "aws_iam_policy_document" "default" {
 }
 
 module "s3_bucket" {
-  source                             = "git::https://github.com/cloudposse/terraform-aws-s3-log-storage.git?ref=tags/0.14.0"
+  source                             = "cloudposse/s3-log-storage/aws"
+  version                            = "0.14.0"
```

Comment:
This version caused me some issue with Terraform 0.14.x since its context uses the [null-label module version 0.19.2](https://github.com/cloudposse/terraform-aws-s3-log-storage/blob/0.14.0/context.tf#L22) which [does not support Terraform 0.14.x](https://github.com/cloudposse/terraform-null-label/blob/0.19.2/versions.tf#L2)